internal/dag/executor/http.go

```diff
@@ -46,6 +46,13 @@ type httpConfig struct {
 	Body    string            `json:"body"`
 	Silent  bool              `json:"silent"`
 	Debug   bool              `json:"debug"`
+	Json    bool              `json:"json"`
+}
+
+type httpJSONResult struct {
+	StatusCode int                 `json:"status_code"`
+	Headers    map[string][]string `json:"headers"`
+	Body       map[string]any      `json:"body"`
```

Comment:
Should we consider changing the `Body` type to `any` because valid JSON can be arrays, primitives, or null, not just objects?

Reply:
You are right, I didn't consider it thoroughly. Thank you for the reminder.